unix-domain-socket/src/main/scala/akka/stream/alpakka/unixdomainsocket/impl/UnixDomainSocketImpl.scala

```diff
@@ -397,7 +405,7 @@ private[unixdomainsocket] abstract class UnixDomainSocketImpl(system: ExtendedAc
                     .andThen {
                       case _ =>
                         try {
-                          file.delete()
+                          path.toFile.delete()
```

Comment:
We could use `Files.delete(path)` to stay consistent with the usage of NIO types.  According to `File.delete` javadocs, `Files.delete` will also throw an `IOException` when applicable, but since we're squelching the exception anyway I guess it's not that important.

Reply:
Will do.